apps/docs/get-started/introduction.mdx

```diff
@@ -4,21 +4,21 @@ title: Welcome 👋
 icon: hand-wave
 ---
 
-Typebot is an fair source chatbot builder. It allows you to create conversational apps/forms (Lead qualification, Customer support, Product launch, User onboarding, AI chats), deploy it on your website or WhatsApp number, and collect results in real-time.
+Typebot is a fair source chatbot builder. It allows you to create conversational apps/forms (Lead qualification, Customer support, Product launch, User onboarding, AI chats), deploy it on your website or WhatsApp number, and collect results in real time.
 
 This is the Typebot documentation. It's a great place to find most answers. Please use the search box to quickly find the answers you're looking for.
 
 <CardGroup cols={2}>
   <Card title="Learn Concepts" icon="graduation-cap" href="./overview">
-    Learn how Typebot works and how to use it to build your first chatbot
+    Learn how Typebot works and how to use it to build your first chatbot.
   </Card>
   <Card title="Deploy your bots" icon="comment" href="/deploy/web/overview">
-    Explore the different ways you can deploy your typebot.
+    Explore the different ways you can deploy your Typebot.
```

Comment:
I used to do the distinction between Typebot the app and typebot the bot you create.

Now we use the word "bot" directly

```suggestion
    Explore the different ways you can deploy your bot.
```